docs/src/main/sphinx/connector/hive.rst

```diff
@@ -668,6 +668,8 @@ features:
 * :ref:`sql-security-operations`, see also :ref:`hive-sql-standard-based-authorization`
 * :ref:`sql-transactions`
 
+If you are migrating from Hive, please refer :doc:`/appendix/from-hive`
```

Comment:
Refer to :doc:`the migration guide </appendix/from-hive>` for practical advice on migrating from Hive to Trino.

Reply:
Fyi ... do not use "please" and other politeness and formal wording in docs..

Reply:
> Fyi ... do not use "please" and other politeness and formal wording in docs..
Got it. Will update this